client/src/components/TextInputComponent/TextInput.stories.js

```diff
@@ -0,0 +1,8 @@
+import React from "react";
+import { storiesOf } from "@storybook/react";
+
+import TextInput from "./TextInput";
+
+storiesOf("TextInput", module).add("Primary", () => (
+  <TextInput label={"Sign up here"}></TextInput>
```

Comment:
You are not passing in a variable here so `"Sign up here"` does not need to be inside `{}`'s. 

Try `label="Sign up here"` instead of `label={"Sign up here"}`